### ISSUE TITLE
Quick Use Hugo-Theme-Jane Error

### ISSUE THREAD

**Issue by MaxdSre** [xianmin/hugo-theme-jane#59]
## Issue 1
Quicking start theme `jane` refer to <https://github.com/xianmin/hugo-theme-jane#quick-start>, it prompts error info while executing `hugo server`

>Building sites … ERROR 2018/05/09 10:59:42 error processing shortcode "theme/shortcodes/gallery.html" for page "post/image-preview.md": template: theme/shortcodes/gallery.html:12:18: executing "theme/shortcodes/gallery.html" at <readDir (print "/sta...>: error calling readDir: Failed to read Directory /static//image/ with error message open /home/ubuntu/myBlog/static/image: no such file or directory
Total in 556 ms
Error: Error building site: logged 1 error(s)

Line **12** in file `myBlog/themes/jane/layouts/shortcodes/gallery.html` is

```
{{- $files := readDir (print "/static/" .) }}
```

It refers to path `~/myBlog/static/image`, but directory `~/myBlog/static/` is empty.

### The solution
The souluiton is add the following command before executing `hugo server`

```bash
cp -R themes/jane/static/image/ static/
```

## Issue 2
The example site content file `myBlog/content/post/image-preview.md` fails to load target image `example.jpg`. On demo site <http://en.xianmin.org/hugo-theme-jane/post/image-preview/>, demo image path is

```markdown
/hugo-theme-jane/image/example.jpg

#line 27
![This is an image in `static/image` folder.](/hugo-theme-jane/image/example.jpg)
```

### The solution
After testing, changing file path to  `/image/example.jpg` will fix the issue.

```markdown
#line 27
![This is an image in `static/image` folder.](/image/example.jpg)
```
Image page link

```
http://localhost:1313/image/example.jpg
```

**Comment by xianmin** [comment]
Thanks for your feedback! Yes, these two issues are my mistakes.

About issue 2, There is a tricky problem.

The `exampleSite/content/` is not just the exapmle content, but also the demo site content.

The problem is the demo site config `baseURL = "http://localhost:1313/hugo-theme-jane/"` .

If I use `![This is an image in static/image folder.](/image/example.jpg)` .
It’s generate `<img src="/image/example.jpg" alt="This is an image in "static/image" folder.">` 
In other words, image link is `http://localhost:1313/image/example.jpg` .
But, in demo site, the image link should be `http://localhost:1313/hugo-theme-jane/image/example.jpg` .

I only considered the generation of the demo site, but did not think of the problem of `quick start` .
I can not find best answer to this problem. Perhaps remove the `image-preview.md` is the best way.
